common/extprofile.h

```diff
@@ -56,6 +56,7 @@ struct ExtendedProfile_Struct {
 	int32				mercState; /* Not Used */
 	uint32				last_invsnapshot_time; /* Used */
 	uint32				next_invsnapshot_time; /* Used */
+	bool 				exp_enabled;
```

Comment:
There's really no good reason why this should live here. This can simply be a private member on `Client` and the setters/getters can mutate it 

eg

```cpp
private:
bool m_exp_enabled = {};
```

Reply:
Reason was we don’t pass client object to save method.